coderd/wsbuilder/wsbuilder_test.go

```diff
@@ -960,6 +961,12 @@ func withInactiveVersion(params []database.TemplateVersionParameter) func(mTx *d
 	}
 }
 
+func withTemplateVersionPreset(presetID uuid.UUID) func(mTx *dbmock.MockStore) {
+	return func(mTx *dbmock.MockStore) {
+		mTx.EXPECT().GetPresetParametersByPresetID(gomock.Any(), presetID).Return(nil, nil)
```

Comment:
I don't understand the value of returning `nil, nil` here; this won't execute your new codepath in `findNewBuildParameterValue`. Can you help me understand pls?